parsers/MK_wrapper.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime, timedelta
+from logging import Logger, getLogger
+from typing import List, Optional, Union
+
+from requests import Session
+
```

Comment:
we'll have to import a timezone module (pytz or datetime.timezone)

Reply:
Maybe we can update to Python 3.9+ soon and use https://docs.python.org/3.11/library/zoneinfo.html#module-zoneinfo

I'll update this in the meantime tho, but I assumed that utcnow() simply returned now() with the UTC datetime but it appears it don't... 😅

Reply:
yeah it's really weird ahah